### PR TITLE
Unit tests 

### PR DESCRIPTION
**Pull request by PoweRGbg**

### CODE REVIEW

**Comment by PoweRGbg** [comment]
@MilosKozak  any idea why Travis fails on an unchanged file ?

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRS/comm/DanaRS_Packet_Bolus_Set_Step_Bolus_Start.java [hunk 1]
```diff
@@ -27,8 +28,12 @@ public DanaRS_Packet_Bolus_Set_Step_Bolus_Start() {
     public DanaRS_Packet_Bolus_Set_Step_Bolus_Start(double amount, int speed) {
         this();
 
-        // HARDCODED LIMIT
-        amount = MainApp.getConstraintChecker().applyBolusConstraints(new Constraint<>(amount)).value();
+        // HARDCODED LIMIT - if there is one that could be created
+        ConstraintChecker constraintChecker = MainApp.getConstraintChecker();
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRS/comm/DanaRS_Packet_Bolus_Set_Step_Bolus_Start.java]: this code brings nothing more

File: app/src/main/res/values/com_crashlytics_export_strings.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```
Comment (MilosKozak) [inline app/src/main/res/values/com_crashlytics_export_strings.xml]: this should be in commit

**Comment by MilosKozak** [comment]
remove the mocking of treatment plugin

File: app/src/main/res/values/com_crashlytics_export_strings.xml (hunk 2 above)
Comment (PoweRGbg) [reply app/src/main/res/values/com_crashlytics_export_strings.xml]: removed

**Comment by PoweRGbg** [comment]
Wow - coverage increase 2.15 %! Many more to come

File: app/src/test/java/info/nightscout/androidaps/plugins/Careportal/Dialogs/NewNSTreatmentDialogTest.java [hunk 3]
```diff
@@ -52,10 +52,10 @@ public void createNSTreatmentTest() throws Exception {
         data.put("duration", 0);
         data.put("percentage", 110);
         data.put("timeshift", 0);
-        dialog.createNSTreatment(data);
+//        dialog.createNSTreatment(data);
```
Comment (MilosKozak) [inline app/src/test/java/info/nightscout/androidaps/plugins/Careportal/Dialogs/NewNSTreatmentDialogTest.java]: why is this removed?
